test/Cardano/FeeSpec.hs

```diff
@@ -133,7 +139,7 @@ spec = do
             }) (Right $ FeeOutput
             { csInps = [20,20]
             , csOuts = [17,18]
-            , csChngs = [1,2]
+            , csChngs = [2,1]
```

Comment:
This line fixes a spurious test failure.

The order here is affected by an arbitrary choice of which change entry to round up, and which to round down.